applications/FluidTransportApplication/python_scripts/apply_scalar_constraint_function_process.py

```diff
@@ -0,0 +1,39 @@
+import KratosMultiphysics
+import KratosMultiphysics.FluidTransportApplication as KratosFluidTransport
+
+def Factory(settings, Model):
+    if(type(settings) != KratosMultiphysics.Parameters):
+        raise Exception("expected input shall be a Parameters object, encapsulating a json string")
+    return ApplyScalarConstraintFunctionProcess(Model, settings["Parameters"])
+
+## All the python processes should be derived from "python_process"
+
+class ApplyScalarConstraintFunctionProcess(KratosMultiphysics.Process):

```

Comment:
I am not familiar with this app but the functionality you implement here is already in the existing in the `AssignScalarVariableProcess` => you can specify a python function as input

Same applies for your other python-process